hr_attendance_geolocation/tests/test_hr_attendance_geolocation.py

```diff
@@ -15,8 +15,9 @@ def setUp(self):
 
     def test_attendance_geolocation(self):
         # Called from js
-        lat, long = self.location
-        self.employee.with_context(latitude=lat, longitude=long).attendance_manual({})
+        self.employee.with_context(attendance_location=self.location).attendance_manual(
+            "hr_attendance.hr_attendance_action_my_attendances"
+        )
```

Comment:
This should stay as it is in v15
```suggestion
        lat, long = self.location
        self.employee.with_context(latitude=lat, longitude=long).attendance_manual({})
```